tests/vmi_configuration_test.go

```diff
@@ -2942,4 +2945,73 @@ var _ = Describe("[sig-compute]Configurations", func() {
 			}, 1*time.Second)).To(Succeed())
 		})
 	})
+	Context("virt-launcher processes memory usage", func() {
+		It("should be lower than allocated size", func() {
+			By("Starting a VirtualMachineInstance")
+			vmi := tests.NewRandomFedoraVMI()
+			vmi, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
+			Expect(err).ToNot(HaveOccurred())
+			tests.WaitForSuccessfulVMIStart(vmi)
```

Comment:
Should we also stress this fedora to try to max out buffer usages of qemu?

Reply:
Yes, good idea. What I used for manual testing triggered oops/panics, so I chose not to include it in the functest :)
However, I assume that's a task `stress-ng` can handle correctly, I'll give it a shot!

Reply:
I think we still have some migration tests which create stress in the VMs. It did not work too great and I was glad that we were able to reduce these tests, but you should find some tests which use it already.

Reply:
Yeah, those were trying to slow down migrations by stressing the guest, not the best idea.
In this case we don't want to actually create stress but just touch all RAM, which it should be able to do.
However, no matter what method we use, if all RAM is in-use that'll at least trigger the guests OOM killer and make a mess, unless... we add swap to the guest? Worth a try.

Reply:
Maybe as a follow-up?

Reply:
I would personally be up for the swap :+1: